docs/integrations/partykit.md

```diff
@@ -0,0 +1,187 @@
+
+```js
+const f = createInstance({
+  datafileUrl: DATAFILE_URL,
+  onReady: () => console.log("SDK has initialized"),
+
+  refreshInterval: 60, // every 60 seconds
+  onRefresh: () => console.log("SDK has fetched the datafile again"),
+});
+```
+
+What it does internally is call the `refresh()` method of the SDK periodically, which we could also call manually ourselves:
+
+```js
+f.refresh();
+```
+
+However, this solution still means that our application will be using the old datafile content as configuration for theoretically maximum 60 seconds before it fetches the latest configuration again.
+
+## Enter WebSocket
+
+[WebSocket](https://developer.mozilla.org/en-US/docs/Web/API/WebSocket) is a communication protocol that provides full-duplex communication channels over a single TCP connection.
+
+By full-duplex, it means both the client and server can send messages to each other independently at the same time by keeping the connection alive.
+
+Since WebSocket is a great way to keep a connection alive between the client and server, we can use this protocol to listen to events from a server that can tell us to trigger a new refresh as soon as there has been any new updates in our Featurevisor project (the Git repository).
+
+## What is PartyKit?
+
+[PartyKit]() is an open source deployment platform for AI agents, multiplayer and local-first apps, games, and websites.
+
+It can help us create a new realtime service that we can send messages to from our CI/CD pipeline whenever there are new changes in our Featurevisor project, and then listen to those messages in our application(s) to trigger a new refresh via our SDK instance.
+
+## Visualizing the whole flow in steps
+
+- We have a Featurevisor project in a Git repository
+- We have a CI/CD pipeline that builds our datafiles and deploys them to a CDN
+  - The CI/CD pipeline with send a message to our PartyKit server whenever there are new changes in our Featurevisor project
+- Our PartyKit server will receive the message and broadcast it to all connected apps
+- Our application(s) will listen to the message and trigger a new refresh of our SDK instance
+
+Let's start implementing this flow step by step.
+
+## Create a new PartyKit server
+
+We can init a new npm project and install PartyKit:
+
+```
+$ npm install --save partykit@beta
+```
+
+Then we create a new `server.js` file and add the following code:
+
+```js
+// replace with your own secret, and inject via environment variable
+const PARTY_SECRET_VALUE = "party-secret";
+const PARTY_HEADER_NAME = "x-partykit-secret";
+
+// the event type we will be broadcasting to all connected apps
+const REFRESH_TYPE = "refresh";
+
+export default {
+  onConnect(websocket, room) {
+    websocket.addEventListener("message", (event) => {
+      websocket.send("pong");
+    });
+  },
```

Comment:
I think your example would work even if you remove this section (lemme know if it doesn't!)

Reply:
@threepointone: can confirm it works without `onConnect`. removing it.